### DISCUSSION TITLE
Can't disable concealer

### DISCUSSION THREAD

**Discussion by flamendless** [nvim-neorg/neorg#860] [Q&A] [answered]
I have the following config:
```lua
	use {
		"nvim-neorg/neorg",
		config = function()
			require("neorg").setup {
				load = {
					["core.defaults"] = {
						config = {
							disable = {
								"core.concealer",
							}
						}
					},
					["core.concealer"] = {
						config = {
							folds = false,
						},
					},
				},
			}
		end,
		run = ":Neorg sync-parsers",
		requires = "nvim-lua/plenary.nvim",
	}
```
however it does not disable concealer at all.

**Comment by OmiChrome** [comment]
you doesn't

**Reply by flamendless** [reply]
But i want to

**Comment by tamton-aquib** [comment, marked as answer]
you could remove or comment the concealer part from the config:

```lua
use {
	"nvim-neorg/neorg",
	config = function()
		require("neorg").setup {
			load = {
				["core.defaults"] = {
					config = {
						disable = {
							"core.concealer",
						}
					}
				},
				-- ["core.concealer"] = {
					-- config = {
						-- folds = false,
					-- },
				-- },
			},
		}
	end,
	run = ":Neorg sync-parsers",
	requires = "nvim-lua/plenary.nvim",
}
```

or am i missing something? 🤔

**Reply by OmiChrome** [reply]
set concealevel=0

**Reply by flamendless** [reply]
> you could remove or comment the concealer part from the config:
> 
> ```lua
> use {
> 	"nvim-neorg/neorg",
> 	config = function()
> 		require("neorg").setup {
> 			load = {
> 				["core.defaults"] = {
> 					config = {
> 						disable = {
> 							"core.concealer",
> 						}
> 					}
> 				},
> 				-- ["core.concealer"] = {
> 					-- config = {
> 						-- folds = false,
> 					-- },
> 				-- },
> 			},
> 		}
> 	end,
> 	run = ":Neorg sync-parsers",
> 	requires = "nvim-lua/plenary.nvim",
> }
> ```
> 
> or am i missing something? 🤔

This works! thanks!
